lib/osf-components/addon/components/contributors/card/readonly/template.hbs

```diff
@@ -0,0 +1,45 @@
+<div data-test-contributor-card ocal-class='CardContainer'>
+    <div data-test-contributor-card-main local-class='MainSection'>
+        <span local-class='CardSection'>
+            <img
+                data-test-contributor-gravatar
+                class='m-l-xs'
+                src={{@contributor.users.links.profile_image}}
+                alt={{@contributor.users.fullname}}
+                height='30'
+                width='30'
+            >
+            <OsfLink
+                data-test-contributor-link
+                data-analytics-name='View user'
+                @route='guid-user'
+                @models={{array @contributor.id}}
+            >
+                {{@contributor.fullName}}
```

Comment:
```suggestion
                {{@contributor.users.fullName}}
```
Same thing here. This is a bit weird: although we have `fullName` property in the contributor model, our API doesn't really serialize it. So for now we have to get the `fullName` from the automatically embedded `users` relationship.